app/src/main/java/com/duckduckgo/app/statistics/api/StatisticsRequester.kt

```diff
@@ -39,19 +44,20 @@ class StatisticsRequester(private val store: StatisticsDataStore, private val se
             return
         }
 
+        val variant = variantManager.getVariant()
+
         service.atb(ParamValue.appVersion)
             .subscribeOn(Schedulers.io())
             .flatMap {
-                store.atb = it.versionWithVariant
-                store.retentionAtb = it.version
-                service.exti(it.versionWithVariant, ParamValue.appVersion)
+                val fullAtb = it.version + variant.key
```

Comment:
The variant often needs to be interrogated independently of the atb, why store them together? I would suggest storing each independently and using a convenience method on the store e.g `atbWithVariant` to retrieve them together when needed.